findme-location-riddles/tests/test_location_riddle_service.py

```diff
@@ -91,7 +92,7 @@ def test_rate_location_riddle(self):
             "mock_location_riddle_id", "mock_user2_id", 3
```

Comment:
there is still some inconsistency in the tests, lets remove all references to `user_id`